bin/ddg2logo

```diff
@@ -39,20 +39,26 @@ description = 'ddg2logo: make logo plots for MutateX DDG values'
 epilog = """ddg2logo generates pseudo-logo plots from DDG values.  While actual logo
 plots are based on letter frequency, these are based on the provided MutateX
 scan and are designed to highlight the mutation sites which specifically feature
-a limited number of stabilizing or accepted mutations. For each site, the
-letters of the plot represent the stabilizing mutations and their height is
-proportional to the stabilizing effect (i.e. to the absolute value of the
-respective negative energy). If a certain site has more than a certain number of
-accepted mutations, the respective column can be replaced by a X - this is
+a limited number of stabilizing (accepted) or destabilizing mutations (not accepted). For each site, the
+letters of the plot represent the stabilizing (or destabilizing) mutations and their height is
+proportional to the stabilizing (or destabilizing)  effect (i.e. to the absolute value of the
+respective energy). If a certain site has more than a certain number of
+accepted (or not accepted) mutations, the respective column can be replaced by a X - this is
 controlled by the -x option. Before plotting the following transformations are
 carried out on the data:
 
-* the DDG values >= than threshold (option -T) are brought down to the threshold
+STABILIZING MUTATIONS    
+* the DDG values >= than the stabilizing threshold (option -T) are brought down to the threshold
 * all the DDG values are modified by subtracting the threshold value,
```

Comment:
I don't think we want this step now, basically we want to 

1) filter out mutations according to DDG threshold
2) plot them as they are (without offsetting for threshold)

Same for both types

Reply:
I'm still not sure what I should do here. For example, I just need to move all the values higher (or lower) than the threshold to the threshold value and then plot the others as they are, or do I need to completely discard them?